src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+h2 {
+  all: unset;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  justify-content: space-between;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)